admin/resources/js/components/form/Select.tsx

```diff
@@ -1,17 +1,22 @@
-import * as React from "react";
+import React from "react";
 import { RegisterOptions, useFormContext } from "react-hook-form";
+import get from "lodash/get";
+import { InputWrapper } from "../H2Components/InputWrapper";
 
-interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+export interface SelectProps
+  extends React.SelectHTMLAttributes<HTMLSelectElement> {
   /** HTML id used to identify the element. */
   id: string;
   /** The text for the label associated with the select input. */
   label: string;
   /** A string specifying a name for the input control. */
   name: string;
   /** List of options for the select element. */
-  options: { value: string | number; label: string }[];
+  options: { value: string | number; text: string }[];
```

Comment:
Because on line 57 or so 

{ label } 

Caused a conflict between the label defined globally and the label from the deconstructed object. No biggie though, Ill swap back to label and just not deconstruct the label (so I can reference it by option.label).

Reply:
Fair enough. I prefer your new version, thanks.